src/peft/tuners/lora/bnb.py

```diff
@@ -35,14 +36,17 @@ def __init__(
             r: int = 0,
             lora_alpha: int = 1,
             lora_dropout: float = 0.0,
+            weight_copy: Optional[torch.Tensor] = None,  # pass weight for LoftQ
             **kwargs,
         ) -> None:
             super().__init__()
-            LoraLayer.__init__(self, in_features=base_layer.in_features, out_features=base_layer.out_features)
+            LoraLayer.__init__(
+                self, in_features=base_layer.in_features, out_features=base_layer.out_features, **kwargs
+            )
             self.base_layer = base_layer
 
-            init_lora_weights = kwargs.pop("init_lora_weights", True)
-            self.update_layer(adapter_name, r, lora_alpha, lora_dropout, init_lora_weights)
+            init_lora_weights = kwargs.pop("init_lora_weights", 'zero')
```

Comment:
Let's not add the `"zero"` option. Can we not just have `init_lora_weights={True, False, "loftq"}`?